### PR TITLE
added logo + organized documentation to make it easier for devs to skim thru

### PR DESCRIPTION
**Pull request by coderofsalvation**
see how it looks at https://github.com/coderofsalvation/expressa

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,68 +73,14 @@ A meta property is added to objects which looks like the following.
...
-
-### Implemented JSON databases
-* MongoDB
-* PostgreSQL (using [jsonb](http://www.postgresql.org/docs/9.4/static/datatype-json.html))
-* Text files (using [json-file-store](https://github.com/flosse/json-file-store) and [mongo-query](https://github.com/Automattic/mongo-query))
+## Documentation 
 
-Other JSON capable databases can be added easily.
+* [Usecase: creating a blog collection and post](doc/blogexample.md)
+* [Authentication](doc/permissions.md)
```
Comment (thomas4019) [inline README.md]: This link is the same as below. I think it should be the following instead.
+* [Permissions](doc/authentication.md)

File: README.md [hunk 2]
```diff
@@ -1,128 +1,70 @@
...
-Those with experience in node and express can mix expressa with their own endpoints, since expressa is just a middleware. It allows adding event listeners which can stop requests and/or modify the results, so advanced functionality can be cleanly added.
+Expressa makes it easy to create basic APIs by using [JSON schema](http://json-schema.org):
 
-The [JSON schema](http://json-schema.org) standard format is used for describing the documents in a collection. This makes your schemas portable, so you can easily use them for another app and with other libraries.
+* django-like __admin interface__ for creating collection-__REST__ endpoints and managing permissions 
+* collection schema's can be edited *and* added __through the admin interface__ 
+* re-use collection schema's in your frontend to generate forms
+* easily extendable so you can add complex features as well
+* define collection as [JSON schema](http://json-schema.org) instead of custom code
+* per-collection database storage: *MongoDB*, *PostgreSQL*, or *JSON-files* (userful for version control)
```
Comment (thomas4019) [inline README.md]: s/userful/useful/

File: README.md [hunk 3]
```diff
@@ -1,128 +1,70 @@
...
 
-Those with experience in node and express can mix expressa with their own endpoints, since expressa is just a middleware. It allows adding event listeners which can stop requests and/or modify the results, so advanced functionality can be cleanly added.
+Expressa makes it easy to create basic APIs by using [JSON schema](http://json-schema.org):
 
-The [JSON schema](http://json-schema.org) standard format is used for describing the documents in a collection. This makes your schemas portable, so you can easily use them for another app and with other libraries.
+* django-like __admin interface__ for creating collection-__REST__ endpoints and managing permissions 
+* collection schema's can be edited *and* added __through the admin interface__ 
+* re-use collection schema's in your frontend to generate forms
+* easily extendable so you can add complex features as well
+* define collection as [JSON schema](http://json-schema.org) instead of custom code
```
Comment (thomas4019) [inline README.md]: s/collection/collections

File: README.md [hunk 4]
```diff
@@ -1,128 +1,70 @@
...
---------------------------------
-
-### The main page of the admin app
-![home](https://cloud.githubusercontent.com/assets/406149/15307573/a43b1508-1b91-11e6-8ff3-c8a24dd4efa1.png)
-
---------------------------------
-
-### Creating the "post" collection and its schema
-![post schema 3](https://cloud.githubusercontent.com/assets/406149/15308189/c3612ade-1b97-11e6-8f87-faeee4aba8bd.png)
+* mix-and-mash: easily thrown in other express middleware and endpoints
```
Comment (thomas4019) [inline README.md]: s/thrown/throw/

**Comment by coderofsalvation** [comment]
ok thx, updated the typos

**Pull request merged**